<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>420</width>
     <height>340</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>JSON字段：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>报文格式：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_jsonPrint">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>253</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>隐藏空值</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>显示全部</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>89</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="text">
     <string> 帧头长度</string>
    </property>
    <property name="cursorPosition">
     <number>5</number>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>89</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string> 帧尾长度</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>89</y>
      <width>108</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>  Protobuf帧</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>120</y>
      <width>108</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>  序列化报文</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_headLength">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>72</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_tailLength">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>120</y>
      <width>72</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>说明：解码时截断帧头帧尾字节长度。</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
